plants/species/Avena_sativa_ot3098/Avena_sativa_ot3098_about.md

```diff
@@ -0,0 +1,6 @@
+About *Genus species*
+---------------------
+
+Cultivated oat (*Avena sativa L.*) is an allohexaploid (AACCDD, 2n = 6x = 42) thought to have been domesticated more than 3,000 years ago while growing as a weed in wheat, emmer and barley fields in Anatolia1.
```

Comment:
It looks like this import has included a superscript '1' at the end of the line ("Anatolia1"), presumably for a footnote in the original. Also, a single line break after this sentence will not start a new paragraph - this will just be one big paragraph of text.